_config.yml

```diff
@@ -99,7 +103,7 @@ defaults:
     values:
       category: os_x
 
-command_categories: ["basics", "shells", "scripting", "tools", "editors", "toys", "ubuntu", "os_x"]
+command_categories: ["basics", "caen", "shells", "scripting", "tools", "editors", "toys", "ubuntu", "os_x"]
```

Comment:
It'd be nice to call this section "tutorials", since this isn't really something CAEN specific, even if we are talking about CAEN in the article. [`caenbrew`](https://github.com/arxanas/caenbrew) would be a good counterexample -- a CAEN specific tool which would clearly go into the hypothetical CAEN section.